client/src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,40 @@
+import { Grid, List, ListItem, ListItemText, Paper, Typography } from "@material-ui/core";
+import useStyles from './useStyles';
+
+export default function Profile(): JSX.Element {
+    const classes = useStyles();
+
+    const listText = [
+        "Edit Profile",
+        "Profile photo",
+        "Availability",
+        "Payment",
+        "Security",
+        "Settings",
+    ]
+
+    return(
+        <Grid container className={classes.root} spacing={2}>
```

Comment:
Hi @aaman123 , as discussed on our meeting today, please make a functional sidebar here, which can be used to route to different components under the profile settings page :)

Reply:
sure rajiv !